provisioning/stream/kubernetes/templates/config/config-map.yaml

```diff
@@ -172,7 +172,7 @@ data:
                 # GZIP compression level: 1-9. Validation rules: min=1,max=9
                 level: 1
                 # GZIP implementation: standard, fast, parallel. Validation rules: required,oneof=standard fast parallel
-                implementation: parallel
+                implementation: fast
```

Comment:
Why did we change parallel to fast? Only reason i see is to reduce number of goroutines used

Reply:
There are `16 threads, on 8 CPU node, ... parallel gzip is alocating a buffer for each thread.

We don't need it by default for multitenant environment (many smaller streams).

And if someone will need it, we can enable it per stream.

Reply:
So in the results we will be able to serve more customers.